src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -17,13 +17,14 @@
 
 
 (defn sqs-ext-client
-  [aws-config & [s3-bucket-name]]
+  [aws-config]
```

Comment:
why did you merged in `s3-bucket-name` here? This creates different meanings for `aws-config` as sometimes it has s3-bucket-name and sometimes it doesn't.  It seems clearer to keep these args separate so that `aws-config` is always:

```
(def aws-config
 {:access-key   "default"
  :secret-key   "default"
  :s3-endpoint  "http://localhost:4566"
  :sqs-endpoint "http://localhost:4566"
  :region       "us-east-2"})
```

We should be consistent with our naming.

Reply:
I would do a multi-arity here with `[aws-config]` and `[aws-config s3-bucket-name]`

Reply:
This made the declaration of the convenience functions in the core ns easier imho. OK, let's go with multi-arity.

Reply:
as discussed, let's rename `aws-config` to `sqs-ext-config`

Reply:
OK, done.